tests/lib/rules/max-lines.js

```diff
@@ -79,17 +77,38 @@ ruleTester.run("max-lines", rule, {
         {
             code: "var xyz;\nvar xyz;\nvar xyz;",
             options: [2],
-            errors: [{ messageId: "exceed", data: { max: 2, actual: 3 } }]
+            errors: [
+                {
+                    messageId: "exceed",
+                    data: { max: 2, actual: 3 },
+                    line: 3,
+                    endLine: 3
```

Comment:
Can we also add `column` and `endColumn` to all invalid tests?

Reply:
Those would be always `1` so I skipped them as that is a known assertion. 

WDYT ?

Reply:
I think it would be good to verify that at least in some tests. Also, `endColumn` shouldn't be `1` for code that doesn't end with a linebreak?